src/extension/wps/wps-core/src/main/java/org/geoserver/wps/gs/GeorectifyConfiguration.java

```diff
@@ -69,7 +66,6 @@
     public GeorectifyConfiguration() {
         GeoServerResourceLoader loader = GeoServerExtensions.bean(GeoServerResourceLoader.class);
         configFile = loader.get(GDAL_CONFIG_FILE);
-        //configFile = new File(GeoserverDataDirectory.getGeoserverDataDirectory(), GDAL_CONFIG_FILE);

```

Comment:
Thanks. I like to remove all commented code I find and I am pleased that you have done so here.